docs/kube-burner.md

```diff
@@ -36,6 +36,15 @@ Each iteration of this workload creates the following objects:
   - 1 deployment holding a client application for the previous database
   - 1 service pointing to the postgresl database
 
+- **max-namespaces**: This workload is a cluster limits focused test which creates maximum possible namespaces across the cluster. This is a namespaced workload, meaning that kube-burner **will createas many namespaces with these objects as the configured job_iterations**. Each namespace will be populated with the following objects.:
+  - 1 deployment holding a postgresql database
+  - 1 deployment holding a client application for the previous database
+  - 1 service pointing to the postgresl database
+
+- **max-services**: This workload is a cluster limits focused test which creates maximum possible services per each namespace. Each namespace will be populated with the following objects.:
```

Comment:
Maybe add that job_iterations defines the number of namespaces?

Reply:
no, this workload only creates one namespace, I've made some modifications in docs to clarify this.

Reply:
Ack, thanks!